client/index.js

```diff
@@ -91,4 +94,10 @@ if ( navigationRoot ) {
 		<CustomerEffortScoreTracksContainer />,
 		root.insertBefore( document.createElement( 'div' ), null )
 	);
+
+	// Used to display CES Survey on the Analytic pages
+	render(
+		<CustomerEffortScoreTracksContainerAnalytics />,
+		root.insertBefore( document.createElement( 'div' ), null )
+	);
```

Comment:
I'm curious to know if there's another place to render this component. This is mostly coming from an aesthetic standpoint, but my leaning is to question the addition of more `render()` calls. 🙃

Reply:
I think we need input from @becdetat  😄

Reply:
I agree, it doesn't seem like something that will scale - it should be possible for third parties to use the CES modal in the same way as we are and adding another container for each use case doesn't seem like it will scale.

I would like to see this done using a store. You can see an example of that in the snackbar implementation itself - there is a `core/notices` store registered in `packages/notices` (actually registered as `core/notices2` since we've forked snackbars from Gutenberg) and consumed in `client/transient-notices`. We could do something similar, registering a `customer-effort-store/surveys` (or similar) store. Then, instead of using an action, getting a survey displayed would be like:

```js
dispatch( 'customer-effort-score/surveys' ).addSurvey(...);
```

Reply:
Thank you @becdetat. I will look into those packages and see if I can change how the component gets rendered.

Reply:
@becdetat would you be OK with adding a new store for CES notices in a new PR? It seems like we could land this one and then refactor the CES notice creation code.

Reply:
@becdetat @jeffstieler I've pushed a new [commit](https://github.com/woocommerce/woocommerce-admin/pull/5594/commits/c679e4733280d56b59c903f2315533582e2a5988) to call the CES survey directly from the analytics components. It's not the cleanest approach, but it's simple and works for now. I would appreciate some feedback on them please :pray: